spec

```diff
@@ -555,7 +555,6 @@ beginseg
     include "build/src/code/z_vr_box.o"
     include "build/src/code/z_vr_box_draw.o"
     include "build/data/code/z_vr_box_draw.data.o"
```

Comment:
What's the story on this data?

Reply:
Looks like z_vr_box_draw actually has no data, and the data labeled as z_vr_box_draw data is actually part of z_sram_NES data. fixing that now